### PR TITLE
[16.0][MIG] fleet_vehicle_log_fuel: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

Changes done:
- [x] Squash administrative commits.
- [x] Set a lot of fields with readonly in some states.
- [x] Add buttons to change status in form view.
- [x] Auto-create a record of the vehicle when the fuel is done.
- [x] Add tests to check all use cases.

@Tecnativa TT48096

### CODE REVIEW

File: fleet_vehicle_log_fuel/README.rst [hunk 1]
```diff
@@ -58,6 +58,10 @@ Contributors
 
 * Miquel Raïch <miquel.raich@forgeflow.com>
 
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Víctor Martínez
```
Comment (pedrobaeza) [inline fleet_vehicle_log_fuel/README.rst:63]: With the previous contribution, we should be here and as co-author, as we rewrite mostly the module.

File: fleet_vehicle_log_fuel/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -1 +1,5 @@
 * Miquel Raïch <miquel.raich@forgeflow.com>
+
```
Comment (pedrobaeza) [inline fleet_vehicle_log_fuel/readme/CONTRIBUTORS.rst]: No need of this empty line

**Comment by victoralmau** [comment]
Changes done. Updated description explaining all changes that have been done.

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM :) Thanks!

**Review by marcelsavegnago — APPROVED** [review]
LGTM

File: fleet_vehicle_log_fuel/README.rst [hunk 3]
```diff
@@ -52,11 +52,16 @@ Authors
...
 * ForgeFlow
+* Tecnativam
 
 Contributors
 ~~~~~~~~~~~~
 
 * Miquel Raïch <miquel.raich@forgeflow.com>
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Víctor Martínez
```
Comment (pedrobaeza) [inline fleet_vehicle_log_fuel/README.rst:63]: Carolina and me are missing here :wink:

File: fleet_vehicle_log_fuel/__manifest__.py [hunk 4]
```diff
@@ -4,17 +4,17 @@
 {
     "name": "Fleet Vehicle Log Fuel",
     "summary": "Add Log Fuels for your vehicles.",
-    "version": "15.0.2.0.0",
-    "category": "Human Resources/Fleet",
-    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "version": "16.0.1.0.0",
+    "category": "Fleet",
+    "author": "ForgeFlow, Tecnativam, Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline fleet_vehicle_log_fuel/__manifest__.py]: We don't plan for now to rebrand our company, hehe

```suggestion
    "author": "ForgeFlow, Tecnativa, Odoo Community Association (OCA)",
```

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**